desktop/notification/notificationtest/gtk.go

```diff
@@ -152,8 +155,13 @@ func (a gtkApi) AddNotification(desktopID, id string, info map[string]dbus.Varia
 }
 
 func (a gtkApi) RemoveNotification(desktopId, id string) *dbus.Error {
-	if a.server.err != nil {
+	dErr := func() *dbus.Error {
```

Comment:
A comment here mentioning that `a.server.Close()` is a locked operation, and that is why we must encapsulate the error retrieval here.